docs/delete-cluster.rst

```diff
@@ -101,6 +146,7 @@ on a per-hour basis. As soon as the cluster is deleted, nothing further will be
 billed for that cluster.
 
 
+.. _AWS Marketplace: https://aws.amazon.com/marketplace/pp/B088P25WGF
```

Comment:
This is the preview (development) product and should not be referenced IMO.
```suggestion
.. _AWS Marketplace: https://aws.amazon.com/marketplace/
```

Reply:
Ah good point!